src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/conf/NSDbConfigs.scala

```diff
@@ -144,21 +170,95 @@ object NSDbConfigs {
               2,
               ConfigDef.Width.MEDIUM,
               NSDB_SHARD_INTERVAL)
-      .define(NSDB_TIMEOUT, Type.STRING, NSDB_TIMEOUT_DEFAULT, Importance.MEDIUM, NSDB_TIMEOUT_DOC)
-      .define(NSDB_SEMANTIC_DELIVERY,
+      .define(NSDB_TIMEOUT,
               Type.STRING,
-              NSDB_SEMANTIC_DELIVERY_DEFAULT,
+              NSDB_TIMEOUT_DEFAULT,
               Importance.MEDIUM,
-              NSDB_SEMANTIC_DELIVERY_DOC)
-      .define(NSDB_AT_LEAST_ONCE_RETRIES,
-              Type.INT,
-              NSDB_AT_LEAST_ONCE_RETRIES_DEFAULT,
-              Importance.LOW,
-              NSDB_AT_LEAST_ONCE_RETRIES_DOC)
-      .define(NSDB_AT_LEAST_ONCE_RETRY_INTERVAL,
-              Type.STRING,
-              NSDB_AT_LEAST_ONCE_RETRY_INTERVAL_DEFAULT,
-              Importance.LOW,
-              NSDB_AT_LEAST_ONCE_RETRY_INTERVAL_DOC)
-
+              NSDB_TIMEOUT_DOC,
+              "Connection",
+              4,
+              ConfigDef.Width.MEDIUM,
+              NSDB_TIMEOUT)
+      .define(
+        NSDB_SEMANTIC_DELIVERY,
+        Type.STRING,
+        NSDB_SEMANTIC_DELIVERY_DEFAULT,
+        ConfigDef.ValidString.in(Constants.AtMostOnce.value, Constants.AtLeastOnce.value),
+        Importance.MEDIUM,
+        NSDB_SEMANTIC_DELIVERY_DOC,
+        "Semantic Delivery",
```

Comment:
non blocking comment: could be maybe used some constants to define the config groups